### ISSUE TITLE
Id field

### ISSUE THREAD

**Issue by JoniDS** [pieces029/shillelagh#28]
Shouldn't it be possible for the Id field to not be AUTO_INCREMENT (so that we can specify the actual id)?

Thanks

**Comment by pieces029** [comment]
Yes, that seems like a reasonable thing to do. I'm trying to kill off these last remaining tickets for a 1.0 release soon, so this will be included in that.

For now you could add another field for your own ids and ignore the AUTO_INCREMENT ids.
